app/views/users/show.html.erb

```diff
@@ -4,10 +4,12 @@
 <p><%= @user.username %></p>
 <p><%= @user.email %></p>
 
+<p>My Movies</p>
 <% @user_movies.each do |movie| %>
-  <%= link_to movie.title, movie_path(movie) %>
+  <li><%= link_to movie.title, movie_path(movie) %></li> 
 <% end %>
 
+<p>My Reviews</p>
 <% @user_reviews.each do |review| %>
-  <%= link_to review.title, movie_path(review.movie) %>
+  <li><%= link_to review.title, movie_path(review.movie) %></li>
 <% end %>
```

Comment:
HTML best practice is to have `<li>` elements belong to a parent `<ul>` or `<ol>` element, inside which they are nested.